Countries Info/GraphQL/Queries.graphql

```diff
@@ -0,0 +1,27 @@
+query MyCountryQuery {
+  Country {
```

Comment:
Lets use file per request, so that we can keep separate query in separate file, for example we can have folder named `Queries` and file inside named `CountryQuery`